src/tpl/layouts/default.nunj

```diff
@@ -21,15 +21,23 @@
         </style>
         <link rel="preload" href="css/main{% if not _dev %}.min{% endif %}.css" as="style" onload="this.rel='stylesheet'">
         <noscript><link rel="stylesheet" href="css/main{% if not _dev %}.min{% endif %}.css"></noscript>
+        <script src="https://ajax.googleapis.com/ajax/libs/webfont/1/webfont.js" async defer></script>
+        <link rel="prefetch" href="https://fonts.googleapis.com/css?family=Roboto:400,700">
         <script>
             // TODO: load minified scripts
             {% include '../../../node_modules/fg-loadcss/src/loadCSS.js' %}
             {% include '../../../node_modules/fg-loadcss/src/cssrelpreload.js' %}
+            WebFontConfig = {
+                google: {
+                    families: ['Roboto:400,700:latin']
+                }
+            };
             {# cookie expired after 24 hours #}
-            document.cookie ='css=1; expires='+(new Date(new Date().getTime() + 86400000)).toGMTString()+'; path=/'
+            document.cookie ='cache=1; expires='+(new Date(new Date().getTime() + 86400000)).toGMTString()+'; path=/'
```

Comment:
I would like to have it as short as possible in this case. It will be sending on each request.